src/main/java/frc/robot/Robot.java

```diff
@@ -108,6 +132,74 @@ public void autonomousInit()
 
     //new TargetingCommandGroup(1, true).start();
     //new TargetingCommandGroup(2, true).start();
+
+    //Sets the flags for the hatch holder and ball intake.
+    if (RobotMap.gamePieceChooser.getSelected().equals("hatch "))
```

Comment:
It's because of how the files are named. If we choose to name the files w/o any spaces then we won't have to deal with them. It's just how i chose to name them before i had written any code.

Reply:
In that case I would 100% make sure to either rename the files or use the String.format to build the string before merging this pull request

Reply:
there are too many paths to try and do a rename. its something to keep in mind for next year

Reply:
You can at least build the string with String.format to add the spaces instead of having them as part of the string literals

Reply:
its easier this way though. 
but in the future we can just not have spaces in the path names

Reply:
It's not much work to refactor it to not have trailing spaces in the strings you're using for comparisons.  Just imagine how long it would take someone else to debug if they accidentally left of the trailing space.

Reply:
that's why we wont do spaces in file names cause it just complicates code

Reply:
As long as we differentiate the words with Capitals at the beginning of each, and determine a consistent naming convention at the beginning it should be fine.
If you come tomorrow i'll show you it working kinda

Reply:
for example LeftCargoRightRocket
meaning robot is starting in the left most position, depositing cargo, onto the right-side of the field, in a rocket

or: MiddleHatchCenterCargoshipLeft
meaning:
robot position: middle
depositing: cargo
into the center of the field
into the cargoship
into the left part

so we would have choosers on shuffleboard for all the different parts of the filename and build the string from that.

thats currently how the code works albeit with spaces causing us to have the trailing spaces